docs/README.md

```diff
@@ -5,56 +5,52 @@
 Vue InstantSearch is the ultimate toolbox for creating instant-search
 experiences using [Vue.js](https://vuejs.org/) and [Algolia](https://www.algolia.com/).
 
-## Algolia demo credentials
```

Comment:
Since the code snippets already embeds those demo credentials, I moved this block down the getting started. So we reach the "live demo" state as soon as possible.